vignettes/adrs.Rmd

```diff
@@ -360,46 +409,30 @@ here and below, in our examples we show these as `ADRS` parameters, but
 they could equally be achieved via `ADSL` dates or `ADEVENT` parameters.
 If you prefer to store as an ADSL date, then the function
 `admiral::derive_var_extreme_dt()` could be used to find the date of
-first `PD` as a variable, rather than as a new parameter record. All the
-parameter derivation functions that use these dates are flexible to
-allow sourcing these from any input source using
-`admiral::date_source()`. See examples below.
+first `PD` as a variable, rather than as a new parameter record.
 
 ## Derive Response Parameter {#rsp}
 
-The next required step is to define the source location for this newly
-derived `PD` date.
+The function `derive_extreme_event()` can then be used to find the date of first
+response. In the below example, the response condition has been defined as `CR`
+or `PR` via the `rsp_y`[^1] event.
```

Comment:
no comment here: only wanted to say that the `[^1]` approach here is very very cool! i like this a lot in how it renders in the vignette and might be worth sharing this at the admiral core dev team sometime as maybe others vignettes could benefit from this. was a new learning for me!

Reply:
It is already used in other vignettes, e.g., the questionnaires vignette. However, most likely not all developers are aware of this feature. I have created an issue to add it to the "Writing Vignette" vignette (https://github.com/pharmaverse/admiraldev/issues/324).